examples/tutorials/tutorial_1_aa_setup.m

```diff
@@ -71,8 +70,9 @@ function tutorial_1_aa_setup()
 % -------------------------------------------------------------------------
 % For BIDS data, point rawdatadir at the top level BIDS directory
 % (i.e., wherever you downloaded ds000114)
-FULLDATAPATH = '/full/path/to/toplevelBIDS';
+FULLDATAPATH =  fullfile(aap.directory_conventions.rawdatadir, 'aa_demo');
```

Comment:
Do we assume the folder to be present, prompt the user (at some stage) to create it, or let aa create it?

Reply:
In the updated aa_downloaddemo, it will be created if it does not exist.